[0.011s] Invoking command in '/home/sycamore/Syca_ws/build/interfaces': /usr/bin/cmake /home/sycamore/Syca_ws/src/interfaces -DCMAKE_INSTALL_PREFIX=/home/sycamore/Syca_ws/install
[0.119s] -- The C compiler identification is GNU 9.4.0
[0.187s] -- The CXX compiler identification is GNU 9.4.0
[0.191s] -- Check for working C compiler: /usr/bin/cc
[0.235s] -- Check for working C compiler: /usr/bin/cc -- works
[0.236s] -- Detecting C compiler ABI info
[0.283s] -- Detecting C compiler ABI info - done
[0.293s] -- Detecting C compile features
[0.293s] -- Detecting C compile features - done
[0.296s] -- Check for working CXX compiler: /usr/bin/c++
[0.357s] -- Check for working CXX compiler: /usr/bin/c++ -- works
[0.358s] -- Detecting CXX compiler ABI info
[0.405s] -- Detecting CXX compiler ABI info - done
[0.413s] -- Detecting CXX compile features
[0.413s] -- Detecting CXX compile features - done
[0.416s] -- Found ament_cmake: 1.1.4 (/home/sycamore/ros2_galactic/ros2-linux/share/ament_cmake/cmake)
[0.433s] -- Found PythonInterp: /usr/bin/python3 (found suitable version "3.8.10", minimum required is "3") 
[0.433s] -- Using PYTHON_EXECUTABLE: /usr/bin/python3
[0.489s] -- Found rosidl_default_generators: 1.1.1 (/home/sycamore/ros2_galactic/ros2-linux/share/rosidl_default_generators/cmake)
[0.491s] -- Using all available rosidl_typesupport_c: rosidl_typesupport_fastrtps_c;rosidl_typesupport_introspection_c
[0.497s] -- Found rosidl_adapter: 2.2.1 (/home/sycamore/ros2_galactic/ros2-linux/share/rosidl_adapter/cmake)
[0.550s] -- Found OpenSSL: /usr/lib/x86_64-linux-gnu/libcrypto.so (found version "1.1.1f")  
[0.567s] -- Found FastRTPS: /home/sycamore/ros2_galactic/ros2-linux/include  
[0.629s] -- Using all available rosidl_typesupport_cpp: rosidl_typesupport_fastrtps_cpp;rosidl_typesupport_introspection_cpp
[0.662s] -- Found PythonInterp: /usr/bin/python3 (found version "3.8.10") 
[0.769s] CMake Error at /home/sycamore/ros2_galactic/ros2-linux/share/rosidl_adapter/cmake/rosidl_adapt_interfaces.cmake:60 (message):
[0.769s]   execute_process(/usr/bin/python3 -m rosidl_adapter --package-name
[0.769s]   interfaces --arguments-file
[0.769s]   /home/sycamore/Syca_ws/build/interfaces/rosidl_adapter__arguments__interfaces.json
[0.769s]   --output-dir
[0.769s]   /home/sycamore/Syca_ws/build/interfaces/rosidl_adapter/interfaces
[0.769s]   --output-file
[0.770s]   /home/sycamore/Syca_ws/build/interfaces/rosidl_adapter/interfaces.idls)
[0.770s]   returned error code 1:
[0.770s] 
[0.770s]   Traceback (most recent call last):
[0.770s] 
[0.770s]     File "/usr/lib/python3.8/runpy.py", line 194, in _run_module_as_main
[0.770s]       return _run_code(code, main_globals, None,
[0.770s]     File "/usr/lib/python3.8/runpy.py", line 87, in _run_code
[0.770s]       exec(code, run_globals)
[0.770s]     File "/home/sycamore/ros2_galactic/ros2-linux/lib/python3.8/site-packages/rosidl_adapter/__main__.py", line 19, in <module>
[0.770s]       sys.exit(main())
[0.770s]     File "/home/sycamore/ros2_galactic/ros2-linux/lib/python3.8/site-packages/rosidl_adapter/main.py", line 53, in main
[0.770s]       abs_idl_file = convert_to_idl(
[0.770s]     File "/home/sycamore/ros2_galactic/ros2-linux/lib/python3.8/site-packages/rosidl_adapter/__init__.py", line 24, in convert_to_idl
[0.770s]       return convert_srv_to_idl(
[0.770s]     File "/home/sycamore/ros2_galactic/ros2-linux/lib/python3.8/site-packages/rosidl_adapter/srv/__init__.py", line 28, in convert_srv_to_idl
[0.770s]       srv = parse_service_string(package_name, input_file.stem, content)
[0.770s]     File "/home/sycamore/ros2_galactic/ros2-linux/lib/python3.8/site-packages/rosidl_adapter/parser.py", line 852, in parse_service_string
[0.770s]       request_message = parse_message_string(
[0.770s]     File "/home/sycamore/ros2_galactic/ros2-linux/lib/python3.8/site-packages/rosidl_adapter/parser.py", line 540, in parse_message_string
[0.771s]       msg = MessageSpecification(pkg_name, msg_name, fields, constants)
[0.771s]     File "/home/sycamore/ros2_galactic/ros2-linux/lib/python3.8/site-packages/rosidl_adapter/parser.py", line 387, in __init__
[0.771s]       self.base_type = BaseType(
[0.771s]     File "/home/sycamore/ros2_galactic/ros2-linux/lib/python3.8/site-packages/rosidl_adapter/parser.py", line 204, in __init__
[0.771s]       raise InvalidResourceName(
[0.771s] 
[0.771s]   rosidl_adapter.parser.InvalidResourceName: 'task_Request' is an invalid
[0.771s]   message name.  It should have the pattern '^[A-Z][A-Za-z0-9]*$'
[0.771s] 
[0.771s] Call Stack (most recent call first):
[0.771s]   /home/sycamore/ros2_galactic/ros2-linux/share/rosidl_cmake/cmake/rosidl_generate_interfaces.cmake:130 (rosidl_adapt_interfaces)
[0.771s]   CMakeLists.txt:19 (rosidl_generate_interfaces)
[0.771s] 
[0.771s] 
[0.771s] -- Configuring incomplete, errors occurred!
[0.771s] See also "/home/sycamore/Syca_ws/build/interfaces/CMakeFiles/CMakeOutput.log".
[0.776s] Invoked command in '/home/sycamore/Syca_ws/build/interfaces' returned '1': /usr/bin/cmake /home/sycamore/Syca_ws/src/interfaces -DCMAKE_INSTALL_PREFIX=/home/sycamore/Syca_ws/install
